<!-- scan2pc.launch -->
<launch>
    <arg name="node_name" default="scan2pc" />
    <arg name="scan_name" default="/scan" />
    <arg name="scan_points_name" default="/scan_points" />

    <node name="$(arg node_name)" pkg="als_ros" type="scan2pc" output="log" >
        <param name="scan_name" value="$(arg scan_name)" type="string" />
        <param name="pc_name" value="$(arg scan_points_name)" type="string" />
    </node >
</launch>
